### ISSUE TITLE
Scatter plots do not draw points in the right locations

### ISSUE THREAD

**Issue by chipkent** [deephaven/web-client-ui#1046]
```python
from deephaven import time_table
from deephaven.plot.figure import Figure
from deephaven.plot import PlotStyle
from deephaven.plot import Color, Colors
from deephaven.plot import font_family_names, Font, FontStyle, Shape


data = time_table('00:00:01') \
    .update(["Open = i + 10*random()","High = Open + 1", "Low = Open - 1", "Close = Open + 0.5"])

points = data \
    .where("i%5 = 0") \
    .view(["Timestamp", "Point = i%3==0 ? High : Low"])

# broken
plot = Figure() \
    .figure_title("OHLC + Points")\
    .plot_xy(series_name="OHLC", t=data, x="Timestamp", y_high="High", y_low="Low", y="Close") \
    .axes(plot_style=PlotStyle.SCATTER)\
    .plot_xy(series_name="Points", t=points, x="Timestamp", y="Point")\
    .point(shape=Shape.DIAMOND, size=16, label="Big Triangle",color=Colors.BLUE )\
   .show()

# broken
 plot = Figure() \
    .axes(plot_style=PlotStyle.SCATTER)\
    .plot_xy(series_name="Points", t=points, x="Timestamp", y="Point")\
    .point(shape=Shape.DIAMOND, size=16, label="Big Triangle",color=Colors.BLUE )\
   .show()  

# broken
 plot = Figure() \
    .axes(plot_style=PlotStyle.SCATTER)\
    .plot_xy(series_name="Points", t=points, x="Timestamp", y="Point")\
   .show()  

# drawing the line plot works
 plot = Figure() \
    .plot_xy(series_name="Points", t=points, x="Timestamp", y="Point")\
   .show()  
```

<img width="825" alt="image" src="https://user-images.githubusercontent.com/5250374/194113153-9cd2c955-5638-42b3-9cf3-4f703c26d69f.png">

**Comment by dsmmcken** [comment]
https://deephaven.atlassian.net/browse/DH-14666 duplicate issue reported in enterprise

**Comment by mofojed** [comment]
There is a workaround for Chrome on M1 Mac: https://github.com/plotly/plotly.js/issues/5970#issuecomment-1550993702

**Comment by mofojed** [comment]
This looks like it is fixed in plotly.js 2.28: https://github.com/plotly/plotly.js/issues/5970#issuecomment-1880680501

We should update to plotly 2.28

**Comment by dsmmcken** [comment]
Re-test #1730 after 2.28 update as well, possibly related, possibly not.

**Comment by mofojed** [comment]
Seems to already be fixed in plotly.js 2.27, and is fixed in v0.33.0:
Engine Version: 0.33.0-SNAPSHOT
Web UI Version: 0.63.0
Java Version: 11.0.21
Barrage Version: 0.6.0
Browser Name: Chrome 121
OS Name: macOS 10.15.7

<img width="1281" alt="image" src="https://github.com/deephaven/web-client-ui/assets/4505624/6977e0c6-a0f1-488c-a608-4ee606b990c5">

We should still update to plotly 2.29, but this issue appears to already be resolved.
